packages/web/src/scss/components/Collapse/_Collapse.scss

```diff
@@ -0,0 +1,32 @@
+@use 'sass:map';
+@use '../../tools/breakpoint';
+@use 'theme';
+
+.Collapse {
+    position: relative;
+    overflow: hidden;
+    height: 0;
+    transition: height 0.25s cubic-bezier(0.4, 0, 0.2, 1) 0s;
```

Comment:
Did we check these values with the designers?

Reply:
Nope, It wasn't in the description, so I've set this up for now with the expectation that someone will comment and provide a better suggestion. 
I will add it to the theme for now